### PR TITLE
TNT-41939 - Adding scope details characteristics

### PR DESCRIPTION
**Pull request by ninaceban**
https://jira.corp.adobe.com/browse/TNT-41939

### CODE REVIEW

**Review by Aaronius — APPROVED** [review]
Looks good. Like @jfkhoury said, it would probably be good to update the examples.

File: extensions/adobe/experience/decisioning/proposition-interaction-detail.example.1.json [hunk 1]
```diff
@@ -26,6 +26,10 @@
         "xdm:algorithmID": "randomuniform",
         "xdm:trafficType": "noncontextual"
       }
-    ]
+    ],
+    "xdm:characteristics": {
+      "xdm:prop1": "Value 1",
```
Comment (swarna04) [inline extensions/adobe/experience/decisioning/proposition-interaction-detail.example.1.json:31]: I believe we should drop the `xdm:` prefix in the examples here to indicate this property bag carries any key-value pairs.

File: extensions/adobe/experience/decisioning/proposition-interaction-detail.example.2.json [hunk 2]
```diff
@@ -44,6 +44,10 @@
         "xdm:algorithmID": "auto-optimize",
         "xdm:trafficType": "noncontextual"
       }
-    ]
+    ],
+    "xdm:characteristics": {
+      "xdm:prop1": "Value 1",
```
Comment (swarna04) [inline extensions/adobe/experience/decisioning/proposition-interaction-detail.example.2.json:49]: Same feedback as above, let's drop the `xdm:` prefix to indicate it doesn't need any specific xdm fields and is a simple container for any data.

File: docs/reference/adobe/experience/decisioning/scope-details.schema.md [hunk 3]
```diff
@@ -81,6 +82,30 @@ The activity that was a requested or chosen for this decision scope. Some scopes
 
 
 
+## xdm:characteristics
+### Scope Details Characteristics
```
Comment (swarna04) [inline docs/reference/adobe/experience/decisioning/scope-details.schema.md:86]: Should we rephrase this to `Scope Characteristics`

File: docs/reference/adobe/experience/decisioning/scope-details.schema.md [hunk 4]
```diff
@@ -81,6 +82,30 @@ The activity that was a requested or chosen for this decision scope. Some scopes
...
+
+Additional properties or attributes belonging to this particular scope details.
```
Comment (swarna04) [inline docs/reference/adobe/experience/decisioning/scope-details.schema.md:88]: How about - `Additional properties or attributes belonging to this particular decision scope.`

**Review by swarna04 — CHANGES_REQUESTED** [review]
Looks good @ninaceban , provided a few minor suggestions here.

File: extensions/adobe/experience/decisioning/proposition-interaction-detail.example.2.json (hunk 2 above)
Comment (swarna04) [reply extensions/adobe/experience/decisioning/proposition-interaction-detail.example.2.json:49]: Sharing an example for option (item) characteristics here for your reference. https://github.com/adobe/xdm/blob/master/docs/reference/adobe/experience/decisioning/option-detail.schema.md#decision-option-details-example

File: docs/reference/adobe/experience/decisioning/scope-details.schema.json [hunk 5]
```diff
@@ -55,6 +55,16 @@
                     "title": "Selected Experience Option",
                     "description": "The experience that was selected when the decision was made.",
                     "$ref": "#/definitions/experience-option-details"
+                },
+                "xdm:characteristics": {
+                    "title": "Scope Details Characteristics",
```
Comment (swarna04) [inline docs/reference/adobe/experience/decisioning/scope-details.schema.json:60]: Here too - `Scope Characteristics`

File: docs/reference/adobe/experience/decisioning/scope-details.schema.md (hunk 4 above)
Comment (cmathis) [reply docs/reference/adobe/experience/decisioning/scope-details.schema.md:88]: Correct. The key's do not need to be namespaced unless you want it that way.

File: docs/reference/adobe/experience/decisioning/scope-details.schema.json (hunk 5 above)
Comment (ninaceban) [reply docs/reference/adobe/experience/decisioning/scope-details.schema.json:60]: ok, thank you

File: docs/reference/adobe/experience/decisioning/scope-details.schema.md (hunk 3 above)
Comment (ninaceban) [reply docs/reference/adobe/experience/decisioning/scope-details.schema.md]: ok, thank you

File: extensions/adobe/experience/decisioning/proposition-interaction-detail.example.2.json (hunk 2 above)
Comment (ninaceban) [reply extensions/adobe/experience/decisioning/proposition-interaction-detail.example.2.json]: ok, thank you

File: extensions/adobe/experience/decisioning/proposition-interaction-detail.example.1.json (hunk 1 above)
Comment (ninaceban) [reply extensions/adobe/experience/decisioning/proposition-interaction-detail.example.1.json]: ok, thank you

File: docs/reference/adobe/experience/decisioning/scope-details.schema.json [hunk 6]
```diff
@@ -55,6 +55,16 @@
                     "title": "Selected Experience Option",
                     "description": "The experience that was selected when the decision was made.",
                     "$ref": "#/definitions/experience-option-details"
+                },
+                "xdm:characteristics": {
```
Comment (prabhum2) [inline docs/reference/adobe/experience/decisioning/scope-details.schema.json:59]: DO you have list of specific keys that you want to populate in this map type ?

Comment (ninaceban) [reply docs/reference/adobe/experience/decisioning/scope-details.schema.json:59]: currently we are thinking of `tnta` and `state` we should still decide on naming, but we would like to use this object to send various details related to Target, ODS and future AJO

File: docs/reference/adobe/experience/decisioning/scope-details.schema.md (hunk 4 above)
Comment (ninaceban) [reply docs/reference/adobe/experience/decisioning/scope-details.schema.md]: thank you
